src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -3344,19 +3344,27 @@ namespace WaterThermalTanks {
             }
 
             Tank.Nodes = DataIPShortCuts::rNumericArgs(32);
+            int specifiedNodes = 0;
             Tank.AdditionalCond = DataIPShortCuts::rNumericArgs(33);
 
             Tank.AdditionalLossCoeff.allocate(Tank.Nodes);
             Tank.AdditionalLossCoeff = 0.0;
-            for (int NodeNum = 1; NodeNum <= Tank.Nodes; ++NodeNum) {
-                if (NumNums > 32 + NodeNum) {
-                    Tank.AdditionalLossCoeff(NodeNum) = DataIPShortCuts::rNumericArgs(33 + NodeNum);
+            for (int NodeNum = 1; NodeNum <= 12; ++NodeNum) {
+                int index = 33 + NodeNum;
+                if (NumNums >= index) {
+                    if (NodeNum <= Tank.Nodes) {
+                        ++specifiedNodes;
+                        Tank.AdditionalLossCoeff(NodeNum) = DataIPShortCuts::rNumericArgs(index);
+                    } else if (!DataIPShortCuts::lNumericFieldBlanks(index) && (DataIPShortCuts::rNumericArgs(index) != 0)) {
+                        // If either blank, or zero (default), then do not warn
+                        ++specifiedNodes;
+                    }
                 } else {
                     break;
                 }
             }
 
-            if (NumNums > 33 + Tank.Nodes) {
+            if (specifiedNodes > Tank.Nodes) {
```

Comment:
Fix is here: if extra nodes are blank, or zero (the default value, which would be filled by IDFEditor or OpenStudio if you use the fields are the loss coefficients), then do not warn.